regulations/templates/regulations/preamble-base.html

```diff
@@ -6,7 +6,7 @@
 
 {% block sidebar-icons %}
 <li>
-  <a href="#table-of-contents" id="menu-link" class="toc-nav-link current" title="Preamble Table of Contents">Pre</a>
+  <a href="#table-of-contents" id="menu-link" class="toc-nav-link" title="Preamble Table of Contents">Pre</a>
```

Comment:
We probably want to calculate this dynamically when rendering the template, but this is a good start.